packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/EmbeddedRealmObjectTests.kt

```diff
@@ -59,6 +63,44 @@ class EmbeddedRealmObjectTests {
         PlatformUtils.deleteTempDir(tmpDir)
     }
 
+    @Test
+    fun parent() {
+        realm.writeBlocking {
+            val parent = copyToRealm(
+                EmbeddedParent().apply {
+                    child = EmbeddedChild()
+                }
+            )
+            parent.child!!.let { child ->
+                child.parent<EmbeddedParent>().let { parent: EmbeddedParent ->
+                    assertNotNull(parent)
```

Comment:
Maybe also check that the parent is actually the right one, perhaps by asserting on some data set on the parent.